python-shaptools.spec

```diff
@@ -30,9 +30,11 @@ Group:          Development/Languages/Python
 Url:            https://github.com/SUSE/shaptools
 Source:         %{name}-%{version}.tar.gz
 %if %{with test}
-BuildRequires:  %{python_module mock}
 BuildRequires:  %{python_module pytest}
 %endif
+%ifpython2
```

Comment:
Can we add a space after the `if` statement, for readability purpose

Reply:
I'm not quite clear on the macro usage here, but I have lots of examples of working spec files with "%ifpython2" in them in my various checkouts, and none with "%if python2", so I think what I already have is correct.